csi/csi.pb.go

```diff
@@ -1,6 +1,6 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.27.1
+// 	protoc-gen-go v1.28.0
```

Comment:
- this doesn't look right
- can you make sure the versions you used to generate supporting files matches the versions mentioned in Makefile?

Reply:
Thanks to the makefile update by @toshipp, it is fixed after the rebase.